core/src/appleMain/kotlin/CBPeripheralCoreBluetoothAdvertisement.kt

```diff
@@ -19,8 +19,8 @@ internal class CBPeripheralCoreBluetoothAdvertisement(
     internal val cbPeripheral: CBPeripheral,
 ) : CoreBluetoothAdvertisement {
 
-    override val identifier: NSUUID
-        get() = cbPeripheral.identifier
+    override val identifier: Uuid
```

Comment:
To be consistent with other platform overrides, let's use `Identifier`.

```suggestion
    override val identifier: Identifier
```

Reply:
I used Uuid because the Peripheral is in that way.

https://github.com/JuulLabs/kable/blob/07e7e96a615359b04e7817b6d8a549426d862c03/core/src/appleMain/kotlin/Peripheral.kt#L125

So, I think interesting change the peripheral as well.

Reply:
Ya, not having the `Peripheral` consistent was an oversight. I'll update the `Peripheral` to be more consistent in a separate PR.